src/app/donation-start/donation-start-form/donation-start-form-new.component.html

```diff
@@ -59,8 +62,8 @@
           </mat-checkbox>
         </div>
 
-        <div *ngIf="!campaign.feePercentage && creditPenceToUse === 0">
-          <mat-form-field appearance="outline">
+      <div *ngIf="!campaign.feePercentage && creditPenceToUse === 0" style="text-align: center; margin-bottom: 50px;">
+            <!-- <mat-form-field appearance="outline">
```

Comment:
```suggestion
            <!-- We may want to bring something like the below back to test a/b test against the slider, although using our
                   custom select component instead of mat-select.
            <mat-form-field appearance="outline">
```